site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -875,6 +901,128 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
 * `Always` - Always append a unique suffix even when the requested username is not in use.
 * `OnCollision` - Only append a unique suffix when the requested username is in use.
 
+[field]#tenant.webAuthnConfiguration.bootstrapWorkflow.authenticatorAttachmentPreference# [type]#[String]# [optional]#Optional# [default]#defaults to `crossPlatform`# [since]#Available since 1.41.0#::
+Determines the authenticator attachment requirement for WebAuthn passkey registration when using the bootstrap workflow. The possible values are:
++
+* `any` - An authenticator with any attachment modality will be allowed during registration.
+* `crossPlatform` - Only authenticators with the `cross-platform` attachment modality will be allowed during registration. These are also referred to as "roaming" authenticators.
+* `platform` - Only authenticators with the `platform` attachment modality will be allowed during registration.
+
++
+The recommended value for the bootstrap workflow is `crossPlatform` (when supported).
+
++
+:premium_feature: WebAuthn
```

Comment:
Should I use `premium_feature` or `advanced_feature` to describe WebAuthn in general? Premium says that a "paid version" is required. Advanced says that "A Essentials or Enterprise" edition is required.

Reply:
@spwitt "Advanced" WebAuthN is Essentials and Ent. for now - https://fusionauth.io/pricing

Reply:
I was wondering because I didn't see any uses of that include. I'm not sure if we are phasing it out or it's just getting started.

Reply:
You were right. Updated to use `advanced_feature` text.